### ISSUE TITLE
Exception on activity creation?

### ISSUE THREAD

**Issue by rosenpin** [HeinrichReimer/android-issue-reporter#36]
I'm trying to implement the issue reporter screen in my app.
Here's my code:
```

public class ReporterActivity extends IssueReporterActivity {
    //Where should the issues go?
    //(http://github.com/username/repository)
    @Override
    public GithubTarget getTarget() {
        return new GithubTarget("rosenpin", "AlwaysOnDisplayAmoled");
    }

    //[Optional] Auth token to open issues if users don't have a GitHub account
    //You can register a bot account on GitHub and copy ist OAuth2 token here.
    @Override
    public String getGuestToken() {
        return "MY_CODE";
    }

    //[Optional] Include other relevant info in the bug report (like custom variables)
    @Override
    public void onSaveExtraInfo(ExtraInfo extraInfo) {
        extraInfo.put("Test 1", "Example string");
        extraInfo.put("Test 2", true);
    }
}

```

And then

`                startActivity(new Intent(getApplicationContext(),ReporterActivity.class));
`

But after I call the startActivity I get this exception in the ReporterActivity class

`                                                                     java.lang.RuntimeException: Unable to start activity ComponentInfo{com.tomer.alwayson/com.tomer.alwayson.Activities.ReporterActivity}: android.view.InflateException: Binary XML file line #37: Binary XML file line #2: Error inflating class android.support.v7.widget.CardView
`

Here's my app gradle file 
```

apply plugin: 'com.android.application'

android {
    compileSdkVersion 24
    buildToolsVersion "24.0.0"

    defaultConfig {
        applicationId "com.tomer.alwayson"
        minSdkVersion 19
        targetSdkVersion 24
        versionCode 107
        versionName "0.87 B"
    }
    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
    }
    compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_7
        targetCompatibility JavaVersion.VERSION_1_7
    }
    lintOptions {
        abortOnError false
    }
}

repositories {
    mavenCentral()
}

dependencies {
    compile 'com.heinrichreimersoftware:android-issue-reporter:1.2.5'
    compile 'com.android.support:appcompat-v7:24.0.0'
    compile 'com.android.support:cardview-v7:24.0.0'
    compile 'com.android.support:design:24.0.0'
    compile 'eu.chainfire:libsuperuser:1.0.0.201602271131'
    compile 'com.github.paolorotolo:appintro:4.0.0'
    compile('de.psdev.licensesdialog:licensesdialog:1.8.1')
    compile 'com.afollestad.material-dialogs:commons:0.8.6.0'
    compile project(':lib')
    testCompile 'junit:junit:4.12'
}

```

And this is my project top level gradle file
```

buildscript {
    repositories {
        jcenter()
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:2.1.2'
    }
}

allprojects {
    repositories {
        jcenter()
        maven { url "https://jitpack.io" }
    }
}

task clean(type: Delete) {
    delete rootProject.buildDir
}

```

**Comment by HeinrichReimer** [comment]
I'll look into this when I'm at home later today.

**Comment by HeinrichReimer** [comment]
Can you post the detaiuled stack trace here? It's especially interesting what the log says after `Error inflating class android.support.v7.widget.CardView `.

**Comment by rosenpin** [comment]
Sure, here is the full log
```

          java.lang.RuntimeException: Unable to start activity ComponentInfo{com.tomer.alwayson/com.tomer.alwayson.Activities.ReporterActivity}: android.view.InflateException: Binary XML file line #37: Binary XML file line #2: Error inflating class android.support.v7.widget.CardView
                                                                        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2416)
                                                                        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476)
                                                                        at android.app.ActivityThread.-wrap11(ActivityThread.java)
                                                                        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344)
                                                                        at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                        at android.os.Looper.loop(Looper.java:148)
                                                                        at android.app.ActivityThread.main(ActivityThread.java:5422)
                                                                        at java.lang.reflect.Method.invoke(Native Method)
                                                                        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
                                                                        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
                                                                     Caused by: android.view.InflateException: Binary XML file line #37: Binary XML file line #2: Error inflating class android.support.v7.widget.CardView
                                                                        at android.view.LayoutInflater.inflate(LayoutInflater.java:539)
                                                                        at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
                                                                        at android.view.LayoutInflater.inflate(LayoutInflater.java:374)
                                                                        at android.support.v7.app.AppCompatDelegateImplV7.setContentView(AppCompatDelegateImplV7.java:280)
                                                                        at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:140)
                                                                        at com.heinrichreimersoftware.androidissuereporter.IssueReporterActivity.onCreate(IssueReporterActivity.java:91)
                                                                        at android.app.Activity.performCreate(Activity.java:6251)
                                                                        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1107)
                                                                        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2369)
                                                                        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476) 
                                                                        at android.app.ActivityThread.-wrap11(ActivityThread.java) 
                                                                        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344) 
                                                                        at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                        at android.os.Looper.loop(Looper.java:148) 
                                                                        at android.app.ActivityThread.main(ActivityThread.java:5422) 
                                                                        at java.lang.reflect.Method.invoke(Native Method) 
                                                                        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
                                                                        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
                                                                     Caused by: android.view.InflateException: Binary XML file line #2: Error inflating class android.support.v7.widget.CardView
                                                                        at android.view.LayoutInflater.createView(LayoutInflater.java:645)
                                                                        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:764)
                                                                        at android.view.LayoutInflater.parseInclude(LayoutInflater.java:941)
                                                                        at android.view.LayoutInflater.rInflate(LayoutInflater.java:831)
                                                                        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
                                                                        at android.view.LayoutInflater.rInflate(LayoutInflater.java:838)
                                                                        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
                                                                        at android.view.LayoutInflater.rInflate(LayoutInflater.java:838)
                                                                        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
                                                                        at android.view.LayoutInflater.rInflate(LayoutInflater.java:838)
                                                                        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
                                                                        at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
                                                                        at android.view.LayoutInflater.inflate(LayoutInflater.java:423) 
                                                                        at android.view.LayoutInflater.inflate(LayoutInflater.java:374) 
                                                                        at android.support.v7.app.AppCompatDelegateImplV7.setContentView(AppCompatDelegateImplV7.java:280) 
                                                                        at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:140) 
                                                                        at com.heinrichreimersoftware.androidissuereporter.IssueReporterActivity.onCreate(IssueReporterActivity.java:91) 
                                                                        at android.app.Activity.performCreate(Activity.java:6251) 
                                                                        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1107) 
                                                                        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2369) 
                                                                        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476) 
                                                                        at android.app.ActivityThread.-wrap11(ActivityThread.java) 
                                                                        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344) 
                                                                        at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                        at android.os.Looper.loop(Looper.java:148) 
                                                                        at android.app.ActivityThread.main(ActivityThread.java:5422) 
                                                                        at java.lang.reflect.Method.invoke(Native Method) 
                                                                        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
                                                                        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
                                                                     Caused by: java.lang.reflect.InvocationTargetException
                                                                        at java.lang.reflect.Constructor.newInstance(Native Method)
                                                                        at android.view.LayoutInflater.createView(LayoutInflater.java:619)
                                                                        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:764) 
                                                                        at android.view.LayoutInflater.parseInclude(LayoutInflater.java:941) 
                                                                        at android.view.LayoutInflater.rInflate(LayoutInflater.java:831) 
                                                                        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798) 
                                                                        at android.view.LayoutInflater.rInflate(LayoutInflater.java:838) 
                                                                        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798) 
                                                                        at android.view.LayoutInflater.rInflate(LayoutInflater.java:838) 
                                                                        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798) 
                                                                        at android.view.LayoutInflater.rInflate(LayoutInflater.java:838) 
                                                                        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798) 
                                                                        at android.view.LayoutInflater.inflate(LayoutInflater.java:515) 
                                                                        at android.view.LayoutInflater.inflate(LayoutInflater.java:423) 
                                                                        at android.view.LayoutInflater.inflate(LayoutInflater.java:374) 
                                                                        at android.support.v7.app.AppCompatDelegateImplV7.setContentView(AppCompatDelegateImplV7.java:280) 
                                                                        at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:140) 
                                                                        at com.heinrichreimersoftware.androidissuereporter.IssueReporterActivity.onCreate(IssueReporterActivity.java:91) 
                                                                        at android.app.Activity.performCreate(Activity.java:6251) 
                                                                        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1107) 
                                                                        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2369) 
                                                                        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476) 
                                                                        at android.app.ActivityThread.-wrap11(ActivityThread.java) 
                                                                        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344) 
                                                                        at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                        at android.os.Looper.loop(Looper.java:148) 
                                                                        at android.app.ActivityThread.main(ActivityThread.java:5422) 
                                                                        at java.lang.reflect.Method.invoke(Native Method) 
                                                                        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
                                                                        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
                                                                     Caused by: java.lang.UnsupportedOperationException: Failed to resolve attribute at index 2: TypedValue{t=0x2/d=0x7f0100c4 a=-1}
                                                                        at android.content.res.TypedArray.getColorStateList(TypedArray.java:482)
                                                                        at android.support.v7.widget.CardView.initialize(CardView.java:220)
                                                                        at android.support.v7.widget.CardView.<init>(CardView.java:116)
                                                                        at java.lang.reflect.Constructor.newInstance(Native Method) 
                                                                        at android.view.LayoutInflater.createView(LayoutInflater.java:619) 
                                                                        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:764) 
                                                                        at android.view.LayoutInflater.parseInclude(LayoutInflater.java:941) 
                                                                        at android.view.LayoutInflater.rInflate(LayoutInflater.java:831) 
                                                                        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798) 
                                                                        at android.view.LayoutInflater.rInflate(LayoutInflater.java:838) 
                                                                        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798) 
                                                                        at android.view.LayoutInflater.rInflate(LayoutInflater.java:838) 
                                                                        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798) 
                                                                        at android.view.LayoutInflater.rInflate(LayoutInflater.java:838) 
                                                                        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798) 
                                                                        at android.view.LayoutInflater.inflate(LayoutInflater.java:515) 
                                                                        at android.view.LayoutInflater.inflate(LayoutInflater.java:423) 
                                                                        at android.view.LayoutInflater.inflate(LayoutInflater.java:374) 
                                                                        at android.support.v7.app.AppCompatDelegateImplV7.setContentView(AppCompatDelegateImplV7.java:280) 
                                                                        at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:140) 
                                                                        at com.heinrichreimersoftware.androidissuereporter.IssueReporterActivity.onCreate(IssueReporterActivity.java:91) 
                                                                        at android.app.Activity.performCreate(Activity.java:6251) 
                                                                        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1107) 
                                                                        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2369) 
                                                                        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476) 
                                                                        at android.app.ActivityThread.-wrap11(ActivityThread.java) 
                                                                        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344) 
                                                                        at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                        at android.os.Looper.loop(Looper.java:148) 
                                                                        at android.app.ActivityThread.main(ActivityThread.java:5422) 
                                                                        at java.lang.reflect.Method.invoke(Native Method) 
                                                                        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
                                                                        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
```

**Comment by HeinrichReimer** [comment]
You have to specify the `cardBackgroundColor` attribute in your reporter theme. I should have stated that in the readme.
```xml
<style ...>
    <item name="cardBackgroundColor">[your color goes here]</item>
</style>
```

**Comment by rosenpin** [comment]
Oh I see, thanks!

**Comment by HeinrichReimer** [comment]
I updated the readme to prevent this issue in future.

**Comment by rosenpin** [comment]
By the way, I had to use `Theme.AppCompat.Light.NoActionBar` as the parent style in order to get it to work properly.

You might want to add it in the README as well.

**Comment by rosenpin** [comment]
Great library BTW, thank you for your work.
I'm using it in my [app](https://github.com/rosenpin/AlwaysOnDisplayAmoled)

**Comment by HeinrichReimer** [comment]
We can create a list of apps that use this library in the readme if you want :grinning:

**Comment by HeinrichReimer** [comment]
BTW you might also like my app intro library [material-intro](https://github.com/HeinrichReimer/material-intro) as a material design alternative to your current intro. It offers cool animations and and adapts the official ["Top User Benefits" onboarding pattern](https://material.google.com/growth-communications/onboarding.html#onboarding-top-user-benefits) 100%.

**Comment by rosenpin** [comment]
I love the idea of creating a list of apps that use this library, sounds great😃

Thanks for linking me to your other library, I'll give it a shot tomorrow :)

**Comment by HeinrichReimer** [comment]
Just notify me when your app is updated with the lib(s).

**Comment by rosenpin** [comment]
I just pushed the update including the issue reporter to production.
I tried the material-intro activity but had issues customizing it to my liking.
I might give it another try later.

**Comment by HeinrichReimer** [comment]
What issues did you have? Anything special? I can help you with that. Your intro looks not too complicated.

**Comment by rosenpin** [comment]
I tried to use the same style from the permission granting simple slide for my special permissions (Draw over other apps, allow system modification) but I couldn't find a way to replicate the 'GRANT PERMISSION' button.

Also, I wanted to disable the next button from within a fragment so users will have to grant permissions before continuing to the next screen, couldn't figure this out either.

I just wish there was a way to I tried to use the same style from the permission granting simple slide for my special permissions (Draw over other apps, allow system modification) but I couldn't find a way to replicate the 'GRANT PERMISSION' button.

Also, I wanted to disable the next button from within a fragment so users will have to grant permissions before continuing to the next screen, couldn't figure this out either.

As I said, I'll give it another shot later, I just have to get other stuff done now
